### ISSUE TITLE
App crashed on pressing Esc key without selecting Color Scheme

### ISSUE THREAD

**Issue by imvikramn** [microsoft/vscode#92315]
Issue Type: <b>Bug</b>

App crashed on pressing Esc key without selecting Color Scheme from the list.

VS Code version: Code 1.43.0 (78a4c91400152c0f27ba4d363eb56d2835f9903a, 2020-03-09T19:34:44.548Z)
OS version: Darwin x64 19.3.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz (4 x 2600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 4, 7|
|Memory (System)|8.00GB (0.47GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
bracket-pair-colorizer-2|Coe|0.0.29
prettier-vscode|esb|3.20.0
remote-containers|ms-|0.106.0
remote-ssh|ms-|0.50.0
remote-ssh-edit|ms-|0.50.0
remote-wsl|ms-|0.42.3
vscode-remote-extensionpack|ms-|0.19.0
material-icon-theme|PKi|4.0.1
remove-empty-lines|use|0.0.6
highlight-matching-tag|vin|0.9.6

(1 theme extensions excluded)

</details>

**Comment by rebornix** [comment]
@imvikramn does it happen all the time when you switch themes?

**Comment by deepak1556** [comment]
I am unable to repro the crash

@imvikramn can you also follow the steps from https://github.com/microsoft/vscode/wiki/Native-Crash-Issues#macos and attach the crash report.

Does it happen when you run with `--disable-extensions` ?

**Comment by imvikramn** [comment]
> @imvikramn does it happen all the time when you switch themes?

It happened only once. I tried to reproduce the issue, it's not happening now.

**Comment by imvikramn** [comment]
> I am unable to repro the crash
> 
> @imvikramn can you also follow the steps from https://github.com/microsoft/vscode/wiki/Native-Crash-Issues#macos and attach the crash report.
> 
> Does it happen when you run with `--disable-extensions` ?

It happened only once. Happened immediately after updating the app. Also the app icon was not visible in the Dock until yesterday while the app is running.

Looks like everything works fine now.

[Electron_2020-03-10-104858_Negup-Mac.txt](https://github.com/microsoft/vscode/files/4316161/Electron_2020-03-10-104858_Negup-Mac.txt)

**Comment by deepak1556** [comment]
Thanks for the trace, symbolicated crashed thread 

```
v8::PropertyDescriptor::configurable() const (in Electron Framework) (api.cc:4126)
electron::WebContentsZoomController::SetZoomFactorOnNavigationIfNeeded(GURL const&) (in Electron Framework) (web_contents_zoom_controller.cc:254)
v8::PropertyDescriptor::enumerable() const (in Electron Framework) (api.cc:4113)
v8::internal::Stats_Runtime_LoadFromSuper(int, unsigned long*, v8::internal::Isolate*) (in Electron Framework) (runtime-classes.cc:720)
v8::internal::AbstractCode::SetStackFrameCache(v8::internal::Handle<v8::internal::AbstractCode>, v8::internal::Handle<v8::internal::SimpleNumberDictionary>) (in Electron Framework) (code.cc:164)
v8::internal::interpreter::BytecodeGenerator::VisitLogicalAndSubExpression(v8::internal::Expression*, v8::internal::interpreter::BytecodeLabels*, int) (in Electron Framework) (bytecode-generator.cc:5711)
v8::internal::RecordMigratedSlotVisitor::RecordMigratedSlot(v8::internal::HeapObject, v8::internal::MaybeObject, unsigned long) (in Electron Framework) (mark-compact.cc:0)
content::MimeSniffingURLLoader::Abort() (in Electron Framework) (mime_sniffing_url_loader.cc:357)
icu_64::TransliteratorIDParser::parseSingleID(icu_64::UnicodeString const&, int&, int, UErrorCode&) (in Electron Framework) (tridpars.cpp:0)
icu_64::TimeZoneFormat::parse(UTimeZoneFormatStyle, icu_64::UnicodeString const&, icu_64::ParsePosition&, int, UTimeZoneFormatTimeType*) const (in Electron Framework) (tzfmt.cpp:915)
icu_64::TimeZoneFormat::formatSpecific(icu_64::TimeZone const&, UTimeZoneNameType, UTimeZoneNameType, double, icu_64::UnicodeString&, UTimeZoneFormatTimeType*) const (in Electron Framework) (tzfmt.cpp:1362)
icu_64::TransliteratorAlias::create(UParseError&, UErrorCode&) (in Electron Framework) (transreg.cpp:122)
icu_64::TransliteratorSpec::reset() (in Electron Framework) (transreg.cpp:313)
i18n_cleanup() (in Electron Framework) (ucln_in.cpp:34)
node::inspector::protocol::DictionaryValue::~DictionaryValue() (in Electron Framework) (Protocol.cpp:538)
proxy_resolver::ProxyResolverFactoryImpl::CreateResolver(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, mojo::PendingReceiver<proxy_resolver::mojom::ProxyResolver>, mojo::PendingRemote<proxy_resolver::mojom::ProxyResolverFactoryRequestClient>) (in Electron Framework) (proxy_resolver_factory_impl.cc:110)
proxy_resolver::MojoProxyResolverV8TracingBindings<proxy_resolver::mojom::ProxyResolverFactoryRequestClient>::~MojoProxyResolverV8TracingBindings() (in Electron Framework) (mojo_proxy_resolver_v8_tracing_bindings.h:33)
(anonymous namespace)::itanium_demangle::AbstractManglingParser<(anonymous namespace)::itanium_demangle::ManglingParser<(anonymous namespace)::DefaultAllocator>, (anonymous namespace)::DefaultAllocator>::parseFunctionType() (in Electron Framework) (ItaniumDemangle.h:2045)
content::SiteIsolationPolicy::GetIsolatedOriginsFromFieldTrial() (in Electron Framework) (__string:225)
(anonymous namespace)::itanium_demangle::FunctionEncoding::printRight((anonymous namespace)::itanium_demangle::OutputStream&) const (in Electron Framework) (ItaniumDemangle.h:809)
__CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
__CFRunLoopDoSource0 + 103
__CFRunLoopDoSources0 + 209
__CFRunLoopRun + 1272
CFRunLoopRunSpecific + 499
RunCurrentEventLoopInMode + 292
ReceiveNextEventCommon + 600
_BlockUntilNextEventMatchingListInModeWithFilter + 64
_DPSNextEvent + 990
-[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1352
-[NSApplication run] + 658
(anonymous namespace)::itanium_demangle::NoexceptSpec::printLeft((anonymous namespace)::itanium_demangle::OutputStream&) const (in Electron Framework) (ItaniumDemangle.h:739)
(anonymous namespace)::itanium_demangle::TemplateArgumentPack::printLeft((anonymous namespace)::itanium_demangle::OutputStream&) const (in Electron Framework) (ItaniumDemangle.h:233)
mojo::ReceiverSetBase<mojo::Receiver<proxy_resolver::mojom::ProxyResolver, mojo::UniquePtrImplRefTraits<proxy_resolver::mojom::ProxyResolver, std::__1::default_delete<proxy_resolver::mojom::ProxyResolver> > >, void>::Entry::Entry(std::__1::unique_ptr<proxy_resolver::mojom::ProxyResolver, std::__1::default_delete<proxy_resolver::mojom::ProxyResolver> >, mojo::PendingReceiver<proxy_resolver::mojom::ProxyResolver>, mojo::ReceiverSetBase<mojo::Receiver<proxy_resolver::mojom::ProxyResolver, mojo::UniquePtrImplRefTraits<proxy_resolver::mojom::ProxyResolver, std::__1::default_delete<proxy_resolver::mojom::ProxyResolver> > >, void>*, unsigned long, bool, scoped_refptr<base::SequencedTaskRunner>) (in Electron Framework) (receiver_set.h:282)
void std::__1::vector<scoped_refptr<url_matcher::URLMatcherConditionSet>, std::__1::allocator<scoped_refptr<url_matcher::URLMatcherConditionSet> > >::__push_back_slow_path<scoped_refptr<url_matcher::URLMatcherConditionSet> >(scoped_refptr<url_matcher::URLMatcherConditionSet>&&) (in Electron Framework) (vector:1626)
v8::Float32Array::CheckCast(v8::Value*) (in Electron Framework) (api.cc:468)
v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>) (in Electron Framework) (counters.h:1187)
v8::Value::NumberValue(v8::Local<v8::Context>) const (in Electron Framework) (tagged-impl.h:55)
viz::mojom::FrameSinkVideoCapturerStubDispatch::Accept(viz::mojom::FrameSinkVideoCapturer*, mojo::Message*) (in Electron Framework) (frame_sink_video_capture.mojom.cc:1308)
viz::mojom::FrameSinkVideoCapturerProxy::Start(mojo::InterfacePtr<viz::mojom::FrameSinkVideoConsumer>) (in Electron Framework) (frame_sink_video_capture.mojom.cc:801)
sh::InitExtensionBehavior(ShBuiltInResources const&, std::__1::map<sh::TExtension, sh::TBehavior, std::__1::less<sh::TExtension>, std::__1::allocator<std::__1::pair<sh::TExtension const, sh::TBehavior> > >&) (in Electron Framework) (Initialize.cpp:38)
viz::mojom::FrameSinkVideoConsumerStubDispatch::Accept(viz::mojom::FrameSinkVideoConsumer*, mojo::Message*) (in Electron Framework) (frame_sink_video_capture.mojom.cc:414)
AtomMain + 84
0x000009e0 (in Electron Framework)
start + 1
```

**Comment by deepak1556** [comment]
Closing this as duplicate of https://github.com/microsoft/vscode/issues/91509 , please ping in the other issue if you see the crash again.
